pkg/util/wellknown.go

```diff
@@ -113,3 +113,12 @@ const (
 	// HTTPGRPCAccessLog sink for the HTTP gRPC access log service
 	HTTPGRPCAccessLog = "envoy.http_grpc_access_log"
 )
+
+// SkipFetchError is the error returned when the cache fetch is short
+// circuited due to the client's version already being up-to-date.
+type SkipFetchError struct{}
```

Comment:
I feel a little strange to put this here, is there anyone else other than `pkg/server/gateway.go` going to need this? Should it be put somewhere in `pkg/cache` actually? cc @kyessenov

Reply:
I too wasn't sure about where this should land. I am not super versed in this codebase, but I did glance around for a spot I thought was fitting. I am certainly open to moving it wherever you all think best. I don't **think** this exact error type would be used anywhere else; but again, I'm deferring to your all's judgement.

Reply:
I think it's fine here given it's an internal implementation detail. This util package needs to be broken down later but it should not be coupled with this change.